workflow-controller/controller-service/src/main/java/io/hyscale/controller/directive/impl/BuildSpecJsonHandler.java

```diff
@@ -47,21 +47,21 @@ public BuildSpec updateObject(BuildSpec buildSpec) {
 		List<Artifact> artifacts = buildSpec.getArtifacts();
 		if (artifacts != null) {
 			artifacts.forEach(artifact -> {
-				artifact.setSource(OperatingSystemHelper.modifyWindowsFileSeparator(artifact.getSource()));
+				artifact.setSource(OperatingSystemUtil.modifyWindowsFileSeparator(artifact.getSource()));
 			});
 			buildSpec.setArtifacts(artifacts);
 		}
 
 		// Config Commands Script
 		String configCmdScript = buildSpec.getConfigCommandsScript();
 		if (StringUtils.isNotBlank(configCmdScript)) {
-			buildSpec.setConfigCommandsScript(OperatingSystemHelper.modifyWindowsFileSeparator(configCmdScript));
+			buildSpec.setConfigCommandsScript(OperatingSystemUtil.modifyWindowsFileSeparator(configCmdScript));
```

Comment:
Can we rename this method modifyWindowsFileSeparator ?

Reply:
Something like updateToUnixFileSeparator?